### PR TITLE
Fixes for media_player.FireTV config language

### PR DESCRIPTION
**Pull request by tmcarr**
**Description:**

Fixes for the config specification as per https://github.com/home-assistant/home-assistant.io/issues/6385

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/media_player.firetv.markdown [hunk 1]
```diff
@@ -58,11 +58,28 @@ media_player:
...
 Configuration variables:
 
-- **host** (*Optional*): The host where `firetv-server` is running. Default is localhost.
-- **port** (*Optional*): The port where `firetv-server` is running. Default is 5556.
-- **device** (*Optional*): The device ID. Defaults to `default`.
-- **name** (*Optional*): The friendly name of the device, default is 'Amazon Fire TV'.
-
+{% configuration %}
+host:
+  description: The host where `firetv-server` is running. Default is localhost
```
Comment (klaasnicolaas) [inline source/_components/media_player.firetv.markdown]: remove default in this rule

File: source/_components/media_player.firetv.markdown [hunk 2]
```diff
@@ -58,11 +58,28 @@ media_player:
...
+  required: false
+  type: string
+  default: localhost
+port:
+  description: The port where `firetv-server` is running. Default is 5556
```
Comment (klaasnicolaas) [inline source/_components/media_player.firetv.markdown]: Same

File: source/_components/media_player.firetv.markdown [hunk 3]
```diff
@@ -58,11 +58,28 @@ media_player:
...
+  required: false
+  type: integer
+  default: 5556
+device:
+  description: The device ID. Defaults to `default`
```
Comment (klaasnicolaas) [inline source/_components/media_player.firetv.markdown]: same

File: source/_components/media_player.firetv.markdown [hunk 4]
```diff
@@ -58,11 +58,28 @@ media_player:
...
+  required: false
+  type: string
+  default: default
+name:
+  description: The friendly name of the device, default is "Amazon Fire TV"
```
Comment (klaasnicolaas) [inline source/_components/media_player.firetv.markdown]: Same

File: source/_components/media_player.firetv.markdown (hunk 1 above)
Comment (cgtobi) [reply source/_components/media_player.firetv.markdown]: And use `""` around the description.

File: source/_components/media_player.firetv.markdown (hunk 2 above)
Comment (cgtobi) [reply source/_components/media_player.firetv.markdown]: Use `""` around the description.

File: source/_components/media_player.firetv.markdown (hunk 3 above)
Comment (cgtobi) [reply source/_components/media_player.firetv.markdown]: Use `""` around the description.

File: source/_components/media_player.firetv.markdown [hunk 5]
```diff
@@ -58,11 +58,28 @@ media_player:
...
+  required: false
+  type: string
+  default: localhost
+port:
+  description: "The port where `firetv-server` is running"
+  required: false
+  type: integer
+  default: 5556
+device:
+  description: "The device ID. Defaults to `default`"
```
Comment (cgtobi) [inline source/_components/media_player.firetv.markdown]: Remove "Defaults to ..."

File: source/_components/media_player.firetv.markdown [hunk 6]
```diff
@@ -58,11 +58,28 @@ media_player:
...
 Configuration variables:
 
-- **host** (*Optional*): The host where `firetv-server` is running. Default is localhost.
-- **port** (*Optional*): The port where `firetv-server` is running. Default is 5556.
-- **device** (*Optional*): The device ID. Defaults to `default`.
-- **name** (*Optional*): The friendly name of the device, default is 'Amazon Fire TV'.
-
+{% configuration %}
+host:
+  description: "The host where `firetv-server` is running"
```
Comment (cgtobi) [inline source/_components/media_player.firetv.markdown]: Please add a `.` at the end of the sentence.

File: source/_components/media_player.firetv.markdown [hunk 7]
```diff
@@ -58,11 +58,28 @@ media_player:
...
+  required: false
+  type: string
+  default: localhost
+port:
+  description: "The port where `firetv-server` is running"
```
Comment (cgtobi) [inline source/_components/media_player.firetv.markdown]: Please add a `.` at the end of the sentence.

File: source/_components/media_player.firetv.markdown [hunk 8]
```diff
@@ -58,11 +58,28 @@ media_player:
...
+  required: false
+  type: integer
+  default: 5556
+device:
+  description: "The device ID. Defaults to `default`"
+  required: false
+  type: string
+  default: default
+name:
+  description: The friendly name of the device
```
Comment (cgtobi) [inline source/_components/media_player.firetv.markdown]: Please add a `.` at the end of the sentence.

File: source/_components/media_player.firetv.markdown [hunk 9]
```diff
@@ -58,11 +58,28 @@ media_player:
 
 Configuration variables:
```
Comment (cgtobi) [inline source/_components/media_player.firetv.markdown]: Remove this line.

Comment (tmcarr) [reply source/_components/media_player.firetv.markdown]: Done....

**Comment by cgtobi** [comment]
Looks like the build fails because of `media_player.universal.markdown`, not because of the changes made in this PR.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @tmcarr!

Happy Hacktoberfest 👍 

../Frenck

**Pull request merged**